spark/src/main/scala/geotrellis/spark/KeyBounds.scala

```diff
@@ -23,10 +25,40 @@ sealed trait Bounds[+A] extends Product with Serializable {
 
   def getOrElse[B >: A](default: => KeyBounds[B]): KeyBounds[B] =
     if (isEmpty) default else this.get
+
+  @inline final def map[B >: A](f: (A, A) => (B, B)): Bounds[B] =
```

Comment:
should map argument be `f: (A, A) => (B, B)` or `f: KeyBounds[A] => KeyBounds[B]` here?